packages/firestore/src/protos/firestore_proto_api.d.ts

```diff
@@ -363,7 +363,7 @@ export declare namespace firestoreV1ApiClientInterfaces {
     booleanValue?: boolean;
     integerValue?: string;
     doubleValue?: number;
-    timestampValue?: string | { seconds: long; nanos: long };
+    timestampValue?: string | { seconds: string; nanos: number };
```

Comment:
Hrm.  I wonder if this should technically be `string|number` ?  I don't think it's required to send `seconds` as a string, but we do to avoid number overflows.

I don't care strongly though.  If it compiles, then it's good enough for now.

Reply:
I think you are right, but that breaks our internal type:

```
// This is a supplement to the generated proto interfaces, which fail to account
// for the fact that a timestamp may be encoded as either a string OR this.
interface TimestampProto {
  seconds?: string;
  nanos?: number;
}
```

That's why I just use strings.